conference/AI/ictai.yml

```diff
@@ -0,0 +1,12 @@
+- title: ICTAI
+  year: 2021
+  id: ictai21
+  description: 33rd International Conference on Tools with Artificial Intellignce
+  link: https://ictai.computer.org/
+  deadline: '2021-06-20 23:59:00'
+  timezone: UTC-7
+  date: November 01-03, 2021
+  place: Washington DC, USA
+  sub: AI
+  rank: C
+  dblp: acml
```

Comment:
I think the dblp is incorrect. It should be ictai instead of acml. Please fix it and pull request again.